### DISCUSSION TITLE
PGM to PandaPower for plotting

### DISCUSSION THREAD

**Discussion by Victor-Stedin** [PowerGridModel/.github#7] [Ideas]
Is it possible to easily use the conversion from power-grid-model-io in order to get the network into PandaPower to use it's plotting functions? Or do i have to create the same network in PandaPower first?

**Comment by nitbharambe** [comment]
Hi @Victor-Stedin ,
Thanks for raising the question! Currently we do not have conversions from `PGM input` to `Pandapower input` to directly plot the PGM data. 

In context of only plotting the `PGM output`:
While we do have conversions from `PGM output` to `Pandapower output`, it is still not possible to plot them as the plotting functions of pandapower needs both `Pandapower input + output` for plotting results.

Hence there is no easy way to use pandapower's plotting function.

**Reply by NielsDirks** [reply]
Dear @nitbharambe,

Is there currently any way to visualise the model? Within the PGM model itself, or by exporting the PGM output to a file that can be imported by Vision again for example? Otherwise, do you expect a way to visualise models any time soon?

**Reply by nitbharambe** [reply]
@NielsDirks, we currently don't have such option. I moved this discussion under `Ideas` so it serves as a feature request within the PGM community.

**Reply by mgovers** [reply]
See also this idea: https://github.com/orgs/PowerGridModel/discussions/14 . It explores the possibility of adding visualization tools for PGM, a.o.. We are actively investigating this.
